src/main/target/KAKUTEF4/target.h

```diff
@@ -126,7 +126,15 @@
 #define RSSI_ADC_CHANNEL            ADC_Channel_11
 
 #define LED_STRIP
-#define LED_STRIP_TIMER         TIM5
+#define WS2811_PIN                      PA0
```

Comment:
I don't think you need these definitions, the resource allocation for the LED strip is already defined by the line in `target.c`.